app/src/main/java/de/westnordost/streetcomplete/quests/crossing_island/AddCrossingIsland.kt

```diff
@@ -0,0 +1,29 @@
+package de.westnordost.streetcomplete.quests.crossing_island
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.mapdata.OverpassMapDataAndGeometryApi
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+class AddCrossingIsland(o: OverpassMapDataAndGeometryApi) : SimpleOverpassQuestType<Boolean>(o) {
+
+    override val tagFilters = """
+        nodes, ways with
+          crossing
```

Comment:
Should be `highway = crossing`?

Reply:
Do islands exists when railway crossings are considered? If not, then yeah, highway=crossing would be more appropriate.

(I personally have not seen islanda on railway crossings, but maybe that's just me)

Reply:
Yeah ... I just took what was in the Issue - taking a closer look, `highway=crossing` is the root tag.